client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -605,6 +601,10 @@ export class BpmnEditor extends CachedComponent {
       };
     }
 
+    if (action === 'elementTemplates.reload') {
```

Comment:
Do we trigger this action anywhere in the app?

Reply:
This action will be triggered by the cloud connect plugin only.